app/views/government_ruleset/show.html.erb

```diff
@@ -1,39 +1,39 @@
-        <table class="govuk-table">
-            <caption class="govuk-table__caption govuk-table__caption--m">Linting Results</caption>
-            <thead class="govuk-table__head">
-            <tr class="govuk-table__row">
-                <th scope="col" class="govuk-table__header">Code</th>
-                <th scope="col" class="govuk-table__header">Line</th>
-                <th scope="col" class="govuk-table__header">Character</th>
-                <th scope="col" class="govuk-table__header">Problem</th>
-                <th scope="col" class="govuk-table__header">Remedy</th>
-            </tr>
-            </thead>
-            <tbody class="govuk-table__body">
-            <% @spectral_results.each do |result| %>
-                <tr class="govuk-table__row">
-                    <th scope="row" class="govuk-table__header"><%= result['code'] %></th>
-                    <td class="govuk-table__cell"><%= result['range']['start']['line'] %></td>
-                    <td class="govuk-table__cell"><%= result['range']['start']['character'] %></td>
-                    <td class="govuk-table__cell"><%= result['message'] %></td>
-                    <td class="govuk-table__cell">
-                        <a href="https://cddo-api-spec-tech-docs.herokuapp.com/api-linting-remediation.html ">Details</a>
-                    </td>
-                </tr>
-            <% end %>
-            </tbody>
-        </table>
+  <div class="govuk-grid-column-two-thirds">
+    <table class="govuk-table">
+      <caption class="govuk-table__caption govuk-table__caption--m">Linting Results</caption>
+      <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Code</th>
+        <th scope="col" class="govuk-table__header">Line</th>
+        <th scope="col" class="govuk-table__header">Character</th>
+        <th scope="col" class="govuk-table__header">Problem</th>
+        <th scope="col" class="govuk-table__header">Remedy</th>
+      </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+      <% @spectral_results.each do |result| %>
+        <tr class="govuk-table__row">
+          <th scope="row" class="govuk-table__header"><%= result['code'] %></th>
+          <td class="govuk-table__cell"><%= result['range']['start']['line'] %></td>
+          <td class="govuk-table__cell"><%= result['range']['start']['character'] %></td>
+          <td class="govuk-table__cell"><%= result['message'] %></td>
+          <td class="govuk-table__cell">
+            <a href="https://cddo-api-spec-tech-docs.herokuapp.com/api-linting-remediation.html ">Details</a>
+          </td>
+        </tr>
+      <% end %>
+      </tbody>
+    </table>
 
-        <a href="
+    <a href="
             <% root_path %>" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button">
-            Select Another Linter
-        </a>
+      Select Another Linter
+    </a>
```

Comment:
There's a [govuk component for this](https://govuk-components.netlify.app/helpers/link/):
```ruby
<%= govuk_link_to "Select Another Linter", root_path %>
```